[15.0] [MIG] l10n_br_hr: Migration to 15.0

rodrigonevest opened the pull request.

mileo committed:
[NEW] Modulo de localicazão dos recursos humanos

Commit:
[IMP] Modificações módulos RH

Conflicts:
	l10n_br_hr_payroll/__openerp__.py

mileo committed:
[NEW] Localização do recusos humanos (hr)

fernandomr committed:
WIP migracao

Conflicts:
	l10n_br_hr_payroll/__init__.py
	l10n_br_hr_payroll/__openerp__.py
	l10n_br_hr_payroll/model/__init__.py
	l10n_br_hr_payroll/model/hr_contract.py

fernandomr committed:
WIP migration to new api

sadamo committed:
Migration of l10n_br_hr module to v8 api

sadamo committed:
Removed unused field n_dependents

sadamo committed:
Fix PEP8

sadamo committed:
FIX views indentation and add README.rst

sadamo committed:
[NEW] Refactor validations and new fields

sadamo committed:
[FIX] Travis errors in module l10n_br_hr

sadamo committed:
Added default values for some fields and fix in dependents validation

sadamo committed:
Adjust view for l10n_br_hr module

sadamo committed:
Move PIS/PASEP validation to l10n_br_base

sadamo committed:
Created security rules and menu items for l10n_br_hr module and removed unused models

sadamo committed:
[FIX] pylint errors

sadamo committed:
Validation in employee dependent CPF field

sadamo committed:
Fields, labels, names in english language

sadamo committed:
Fix CBO code format error

leticiadnsouza committed:
.PO file to translate to portuguese

mileo committed:
[NEW] Add yml tests to l10n_br_hr

sadamo committed:
[NEW] Criação do menu de configuração RAIS

sadamo committed:
[FIX] fields type based on RAIS manual

mileo committed:
[NEW] cnpj_cei in res.company

hendrixcosta committed:
[ADD] Informações bancarias para dependente

Commit:
[FIX] Renomeado ID para Identidade

Commit:
[FIX] Removidos métodos que auto-populavam o campo address_home_id

Commit:
[FIX] Tradução etnias

hendrixcosta committed:
[FIX] Ajustar tabela igual o esocial

Commit:
[FIX] Corrigido tabela hr.ethnicity para equivaler com o o campo racaCor do e-Social

Commit:
[FIX] Conflito

Commit:
[FIX] Ajustes para o e-Social

lfdivino committed:
[FIX] Permissões nas abas da view do funcionário

hendrixcosta committed:
[REF] Melhorias para esocial

lfdivino committed:
[ADD] View dos Dependentes do Funcionário

hendrixcosta committed:
[FIX] Remover tradução errada

mileo committed:
[MIG] Bump module version to 10.0.1.0.0

mileo committed:
[MIG] odoo namespace

mileo committed:
[MIG] odoo imports

mileo committed:
pep8

BiancaBartolomei committed:
[MIG]Remoção de <data> e </data> em xml noupdate=0

BiancaBartolomei committed:
[MIG]Correções de referências quebradas e troca de string por name

BiancaBartolomei committed:
[MIG]Correção de referências

BiancaBartolomei committed:
[MIG]Correção dos testes

BiancaBartolomei committed:
[MIG]Deixar hr independente do hr_contract

BiancaBartolomei committed:
[FIX]Correção da visão do employee

BiancaBartolomei committed:
[FIX]Obrigatoriedade do campo CPF na view

BiancaBartolomei committed:
[ADD]Valida e formata cpf

BiancaBartolomei committed:
[FIX]pep8

BiancaBartolomei committed:
[FIX]country da formatacao do cpf

BiancaBartolomei committed:
[ADD]oe_chatter em outros campos

BiancaBartolomei committed:
[FIX]Simplificação do check_cpf

BiancaBartolomei committed:
[FIX]Simplificação do validate_pis_pasep e renomeação

renatonlima committed:
pep8

OCA-git-bot committed:
[ADD] icon.png

renatonlima committed:
[ADD] pot translation files

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-10.0/l10n-brazil-10.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-10-0/l10n-brazil-10-0-l10n_br_hr/

renatonlima committed:
[ADD] pot file

luismalta committed:
[MIGR] __manifest__

luismalta committed:
[MIG] Refactoring to Python 3

luismalta committed:
[MIG]Remove 'coding: utf-8' from python files

luismalta committed:
[MIG]hr_employee.py

luismalta committed:
[MIG]hr_job.py, l10n_br_hr.py, res_company.py

luismalta committed:
[MIG]Translatable term and indentation in hr_employee.py

luismalta committed:
[MIG]__manifest__: fix version and author strings

luismalta committed:
[MIG]Change view_res_company_form_l10n_br_hr reference

luismalta committed:
[MIG]Migrate tests to python

luismalta committed:
[MIG]Fix comodel_names

luismalta committed:
[MIG]Tests indentation fix

luismalta committed:
[MIG]Change module l10n_br_base.tools to erpbrasil.base.fiscal

luismalta committed:
[MIG]Add external dependencies to erpbrasil.base

luismalta committed:
[MIG]Correct erpbrasil.base version

luismalta committed:
[MIG] Increases test coverage

luismalta committed:
[MIG]Correct E501

luismalta committed:
[MIG]Invalid pis_pasep and cpf tests

luismalta committed:
[MIG]Correct E303 in test

luismalta committed:
[MIG]Migrate features from 8.0

luismalta committed:
[MIG]Models modularization

luismalta committed:
[MIG]Fix pylint

luismalta committed:
[MIG] Translation file

luismalta committed:
[MIG] Upgrade erpbrasil.base version

luismalta committed:
[MIG]Fix external dependencies

luismalta committed:
[MIG] Add models descriptions

luismalta committed:
[MIG] Add related field to employee CPF

mileo committed:
Update hr_employee.py

mileo committed:
[REF] hr_employee

mileo committed:
Update README.rst

mileo committed:
[REM] White space

mileo committed:
[REM] required of cpf

oca-travis committed:
[UPD] Update l10n_br_hr.pot

renatonlima committed:
[REF] Added abstract model for HR data objects

renatonlima committed:
[REF] Removed deprecated attributes

renatonlima committed:
[REF] split data files

renatonlima committed:
Renamed data file

renatonlima committed:
[MIGR] readme files

Commit:
[REM] Model hr.chronic.disease

oca-travis committed:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

Commit:
[ADD] format CPF

Commit:
[FIX] Validate dependent

Commit:
[REF] Update dependent type

Commit:
[REF] Adotando o padrão de modelos abstratos

luismalta committed:
[REF] Running pre-commit on l10n_br_hr

Commit:
[FIX] Flake8

Commit:
[FIX] Utilizando context_today

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

mileo committed:
[REM] unused imports

luismalta committed:
[FIX] Blank line at the end of file

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

Commit:
[ADD] Test for dependent

Commit:
[FIX] Flake8

Commit:
[FIX] Port nationality code to res_country

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

Commit:
[ADD] Demo

Commit:
[REM] Função obsoleta

Commit:
[ADD] Test for hr.employee

Commit:
[FIX] Flake8

Commit:
[ADD] test onchange CPF employee

mileo committed:
[FIX] partner_id of dependent invisible

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

mileo committed:
[REF] missing pre-commit in l10n_br_hr

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

renatonlima committed:
[FIX] l10n_br_hr text

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

mliell committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.6% (279 of 315 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (321 of 321 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (321 of 321 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.6% (324 of 325 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (325 of 325 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-travis committed:
[UPD] Update l10n_br_hr.pot

mileo committed:
[REF] Standard OCA website l10n_br_hr

rvalyi committed:
[IMP] l10n_br_hr: black, isort, prettier

rvalyi committed:
remove useless default @api.multi

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-travis committed:
[UPD] Update l10n_br_hr.pot

rvalyi committed:
fix erp.brasil dependency

marcelsavegnago committed:
[IMP] l10n_br_hr: black, isort, prettier

marcelsavegnago committed:
[IMP] l10n_br_hr: remove view_type

marcelsavegnago committed:
[IMP] l10n_br_hr: change track_visibility by tracking attribute

marcelsavegnago committed:
[FIX] l10n_br_hr: fix _name_search

marcelsavegnago committed:
[FIX] l10n_br_hr: black, isort, prettier

marcelsavegnago committed:
[MIG] l10n_br_hr: Migration to 14.0

marcelsavegnago committed:
[FIX] l10n_br_hr: test

marcelsavegnago committed:
[IMP] l10n_br_hr: add cnpj_cpf on hr.employee

oca-travis committed:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

oca-travis committed:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 82.9% (297 of 358 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

Commit:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

Commit:
[UPD] Update l10n_br_hr.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

mileo committed:
[REF] do not import tests l10n_br_hr

mileo committed:
[REF] Context better using kwargs l10n_br_hr

mileo committed:
[REF] attribute-string-redundant l10n_br_hr

mileo committed:
[REF] xml-deprecated-tree-attribute l10n_br_hr

Commit:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
l10n_br_hr 14.0.1.0.1

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 78.7% (286 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 79.0% (287 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 86.2% (313 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 86.2% (313 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 86.2% (313 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 86.2% (313 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.4% (321 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.9% (323 of 363 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

Commit:
[UPD] Update l10n_br_hr.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.2% (323 of 366 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
[FIX] l10n_br_hr: add group hr.group_hr_user on private brazilian fields

OCA-git-bot committed:
l10n_br_hr 14.0.1.1.0

marcelsavegnago commented:
/ocabot migrate l10n_br_hr

marcelsavegnago commented:
/ocabot migration l10n_br_hr

kaynnan reviewed (approved):
LGTM

rvalyi reviewed (changes requested):
@marcelsavegnago @mileo eu acho que seria a oportunidade de começar a dar uma limpa nesses modulos de hr. Na epoca da v8 que foram feitos talvez o scope da OCA era muito menor, mas hoje é meio que um tiro no pé esse modulo de HR ficar re-criando conceitos que ja tem modulos OCA e todo um eco-sistema crescendo por cima.

Então por examplo:

1. tirar hr.employee.dependent ou herdar de https://github.com/OCA/hr/tree/14.0/hr_employee_relative
2. ? usar esse modulo https://github.com/OCA/hr/tree/14.0/hr_employee_ssn ?
3. tirar hr.educational.attainment e usar o modulo do Odoo CE hr_skills ?

De uma forma geral, tem tb muitos campos cadastrais que são para fazer o eSocial. Mas agora com a technologica XML que temos, a gente consegue trazer esses campos pelos XSD do esocial... Então esta OK de ter alguns campos usuais num modulo hr basicão que não tem a pretensão de fazer ele mesmo a porra toda do esocial e não de ter as dependenciais disso. Porem vale a reflexão se nesse modulo ja não estaria tendo alguns campos que ja tem um uso bem avançado orientado esocial e que ai poderiam ser tirado do modulo e trazido apenas na hora de fazer o modulo do esocial.

Por fim eu consigo dar um feedback sobre isso dos mixins abstratos do esocial em questãoes de uns 10 dias. Eu acho que da para temporisar um pouco o merge ate esse feedback. Mas eu acho que ainda assim os outros pontos que eu levantei valem.

marcelsavegnago commented:
@rvalyi acho valido sim considerar o uso de outros módulos já maduros na OCA. Sobre o eSocial eu não consigo opinar muito agora.

marcelsavegnago commented:
@rvalyi seria melhor na 14 propor um refactor ou trabalhar direto no refactor durante a migracao para 15 ? A principio focariamos em focar na adaptação para usar os módulos da OCA para substituir alguma implantação feita no módulo.

rvalyi commented:
então, sobre usar hr_employee_relative ou hr_skills (teria outros?) a gente poderia nem botar a dependencia (para não fuder com a modularidade do Odoo por conta da localização; um principio direitor do projeto aqui). Ai sei la a gente poderia apenas sugerir no README de instalar esses modulos para ter esse tipo de funcionalidade. E se la na frente tiver modulos mais orientados e-Social, ai a gente iria sim depender desses modulos e fazer os mappings entre os campos dos XSD e do modulos OCA/Odoo que nem a gente faz no l10n_br_nfe para a NFe.

Mas ai na 14, pro usuario nao perder essas informaçoes, até existir tais modulos de eSocial a gente poderia não remover esses campos, apenas botar um comentario que esta deprecated ou algo assim. E ai qdo tiver modulo de esocial ter um script para migrar esses dados talvez. Mas enfim é considerando isso que a gente vé a importancia de acertar o modelo de dados quanto antes que senão la ne frente vai chover de usuarios/curiosos enchando o saco se deixar para acertar depois.

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.2% (323 of 366 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

rvalyi committed:
[REF] simplify transitive dependency

OCA-git-bot committed:
l10n_br_hr 14.0.1.2.0

rvalyi committed:
[REF] faster tests with SavePointCase

OCA-git-bot committed:
l10n_br_hr 14.0.1.2.1

Commit:
[UPD] Update l10n_br_hr.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

Commit:
[UPD] Update l10n_br_hr.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update l10n_br_hr.pot

mbcosta committed:
[REF] l10n_br_hr: Unified IE, CNPJ and CPF validations.

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

mbcosta committed:
[REF] l10n_br_hr: Inform python external dependencies make unnecessary use TRY to import a library.

mbcosta committed:
[REF] l10n_br_hr: Remove empty Lines, necessary for the new version of pre-commit.

mbcosta committed:
[REF] l10n_br_hr: Method Super without parameters, necessary for the new version of pre-commit.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

marcelsavegnago committed:
[IMP] l10n_br_hr: add pis_pasep_date

Commit:
[UPD] Update l10n_br_hr.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

Commit:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.4% (315 of 356 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

douglascstd committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 89.6% (319 of 356 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

Commit:
[UPD] Update l10n_br_hr.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 87.6% (319 of 364 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 87.9% (320 of 364 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.1% (321 of 364 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 88.1% (321 of 364 strings)

Translation: l10n-brazil-14.0/l10n-brazil-14.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_hr/pt_BR/

rvalyi reviewed (approved):
fazendo o merge na 15,0, porem com as ressalvas https://github.com/OCA/l10n-brazil/pull/2437#pullrequestreview-1384358831
que eu acho que devemos considerar se portar o modulo para a v16.

rvalyi commented:
/ocabot merge nobump

marcelsavegnago commented:
@rvalyi estou refazendo a branch e ja mando novamente

marcelsavegnago committed:
[IMP] l10n_br_hr: black, isort, prettier

marcelsavegnago committed:
[MIG] l10n_br_hr: Migration to 15.0

marcelsavegnago commented:
/ocabot merge nobump
